definitions.yaml

```diff
@@ -141,6 +147,12 @@ definitions:
         type: integer
         format: int32
         description: Expiration time (from creation) in hours
+      cn_prefix:
+        type: string
+        description: The common name prefix of the certificate subject.
+      o:
```

Comment:
`certificate_organizations` and `common_name_prefix` works for me. It is self explaining and aligns with the style we use so far.